.prettierrc

```diff
@@ -1,6 +1,15 @@
 {
-	"useTabs": true,
-	"singleQuote": true,
-	"trailingComma": "none",
-	"printWidth": 100
+  "useTabs": false,
```

Comment:
question: I suppose all the whitespace changes are there because if this change? I like the switch to spaces instead of tabs tho!

Reply:
correct!